yt/visualization/plot_container.py

```diff
@@ -257,7 +257,13 @@ def __init__(self, data_source, figure_size, fontsize):
             self.figure_size = float(figure_size[0]), float(figure_size[1])
         else:
             self.figure_size = float(figure_size)
-        font_path = matplotlib.get_data_path() + "/fonts/ttf/STIXGeneral.ttf"
+
+        try:
+            font_path = matplotlib.get_data_path() + "/fonts/ttf/STIXGeneral.ttf"
+        except TypeError:
+            # MPL 3.4.3 returns a PosixPath
+            font_path = str(matplotlib.get_data_path()) + "/fonts/ttf/STIXGeneral.ttf"
```

Comment:
```suggestion
        # MPL >= 3.4.3 returns a Path
        font_path = os.path.join(matplotlib.get_data_path(), "fonts", "ttf", "STIXGeneral.ttf")
```
- using `os.path.join` will not only convert to string, it will also make this line windows-compatible for free !
- the try block is actually not needed since the fallback option is always going to work, so let's go with that one in any case
- the comment needs slight correction: the exact return type of `matplotlib.get_data_path` is not going to be `PosixPath` on windows, (rather a `WindowsPath`, see pathlib's doc https://docs.python.org/3/library/pathlib.html)

Reply:
okay, great, thanks for the ideas.  I'll update.